src/Components/Onboarding/Steps/Artists/ArtistSearchResults.tsx

```diff
@@ -92,47 +86,50 @@ class ArtistSearchResultsContent extends React.Component<RelayProps, null> {
   }
 
   onFollowedArtist(artist: Artist) {
-    commitMutation(this.props.relay.environment, {
-      mutation: graphql`
-        mutation ArtistSearchResultsArtistMutation(
-          $input: FollowArtistInput!
-          $excludedArtistIds: [String]!
-        ) {
-          followArtist(input: $input) {
-            popular_artists(
-              size: 1
-              exclude_followed_artists: true
-              exclude_artist_ids: $excludedArtistIds
-            ) {
-              artists {
-                id
-                _id
-                __id
-                name
-                image {
-                  cropped(width: 100, height: 100) {
-                    url
+    commitMutation<ArtistSearchResultsArtistMutation>(
+      this.props.relay.environment,
+      {
+        mutation: graphql`
+          mutation ArtistSearchResultsArtistMutation(
+            $input: FollowArtistInput!
+            $excludedArtistIds: [String]!
+          ) {
+            followArtist(input: $input) {
+              popular_artists(
+                size: 1
+                exclude_followed_artists: true
+                exclude_artist_ids: $excludedArtistIds
+              ) {
+                artists {
+                  id
+                  _id
+                  __id
+                  name
+                  image {
+                    cropped(width: 100, height: 100) {
+                      url
+                    }
                   }
                 }
               }
-            }
-            artist {
-              __id
-              related {
-                suggested(
-                  first: 1
-                  exclude_followed_artists: true
-                  exclude_artist_ids: $excludedArtistIds
-                ) {
-                  edges {
-                    node {
-                      id
-                      _id
-                      __id
-                      name
-                      image {
-                        cropped(width: 100, height: 100) {
-                          url
+              artist {
+                __id
+                related {
+                  suggested(
+                    first: 1
+                    exclude_followed_artists: true
+                    exclude_artist_ids: $excludedArtistIds
+                  ) {
+                    edges {
+                      node {
+                        id
+                        _id
+                        __id
```

Comment:
Not related to this pr, but 🤔

Reply:
The first two are needed for at least analytics purposes, the latter to work with Relay’s store (and is fetched implicitly anyways)